pandas/_libs/tslibs/timestamps.pyx

```diff
@@ -336,6 +336,23 @@ class Timestamp(_Timestamp):
         """
         return cls(datetime.combine(date, time))
 
+    @classmethod
+    def fromisocalendar(cls, year, week, day):
+        """
+        Timestamp.fromisocalendar(year, week, day)
+
+        Return a new Timestamp corresponding to the
+        ISO calendar date specified by year, week and day.
+        """
+        import pandas.compat as compat
```

Comment:
@jbrockmendel can we not import this at the top? (else we should have these constants in a .pyx somewhere

Reply:
importing this at the top is not an option, but we can either put it in e.g. tslibs.util or from `cpython.version cimport PY_MINOR_VERSION`.

Reply:
right should prob make a standard way of doing this (though i don't think we need to have version checks too often in cython these days)